db/seeds/utils.js

```diff
@@ -1,3 +1,6 @@
+const db = require('../connection')
+const format = require("pg-format")
```

Comment:
As you have used this in a function that will be needed when the api is in production, make sure pg-format is a regular dependancy now. If it is a dev dependancy it will be ignored in hosting (we don't need jest installed on our server when its hosted as we aren't testing it). If it is ignored but needed the build will fail.